java/src/jmri/jmrit/vsdecoder/swing/ManageLocationsAction.java

```diff
@@ -53,51 +66,49 @@ public void actionPerformed(ActionEvent e) {
             listenerLoc = VSDecoderManager.instance().getVSDecoderPreferences().getListenerPosition();
 
             // Handle Reporters
-            ReporterManager rmgr = jmri.InstanceManager.getDefault(jmri.ReporterManager.class);
+            ReporterManager rmgr = jmri.InstanceManager.getDefault(ReporterManager.class);
             Set<Reporter> reporterSet = rmgr.getNamedBeanSet();
             Object[][] reporterTable = new Object[reporterSet.size()][7];
             int i = 0;
             for (Reporter r : reporterSet) {
                 if (r != null) {
                     if (r instanceof PhysicalLocationReporter) {
                         PhysicalLocation p = ((PhysicalLocationReporter) r).getPhysicalLocation();
-                        reporterTable[i][0] = r.getSystemName();
-                        reporterTable[i][1] = r.getDisplayName();
-                        reporterTable[i][2] = true;
-                        reporterTable[i][3] = p.getX();
-                        reporterTable[i][4] = p.getY();
-                        reporterTable[i][5] = p.getZ();
-                        reporterTable[i][6] = p.isTunnel();
+                        reporterTable[i][SYSNAMECOL] = r.getSystemName();
+                        reporterTable[i][USERNAMECOL] = r.getUserName();
```

Comment:
Have you checked the case when user name is null? getUserName() may return null if there is no user name, while getDisplayName() always return a string.

Reply:
Yes, and it works for a ```jmri.Block``` and a ```jmri.Reporter```. Will change it to ```getDisplayName()```. Thanks!